client/style.scss

```diff
@@ -167,6 +167,16 @@ iframe {
 	}
 }
 
+.image-error {
+
+	color: $red !important;
```

Comment:
Is important needed here?
Would prefer not to have it, but sometimes it is not possible with Aurora.

Reply:
Would also probably call `.image-warning` and use yellow instead.